task8/script.js

```diff
@@ -0,0 +1,61 @@
+const documentFragment = document.createDocumentFragment();
+
+const ul = document.getElementById("ul");
+
+const listItem = [
+  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+// ローディング画像を表示
+const insertLoading = () => {
+  const loading = document.getElementById("div");
```

Comment:
Q
Why is it `loading'?

If you name it loading.
```js
const loading = document.getElementById("loading");
```
It's better this way, but...
If it were me, I'd simply use div in this case.(IMO)
```js
const div = document.getElementById("div");
```

Reply:
Does it mean that the ID name and function name should be simple?

commit: cdf434b

Reply:
I was wondering why the HTML IDs and constant names were different, so I asked.
The reason for giving IDs is that they are used in JS.
```js
const loading = document.getElementById("div");
```
It is not clear what the ID is for when you look at it in the HTML, and you can only understand it by looking at the JS code.So whenever anyone sees this code, it is a good idea to name it in such a way that they will know where to use the ID.


And I should have told you in my last comment, but I thought it would be better to add `js-`. I'm sorry I forgot!!!!
```js
const div = document.getElementById("js-div");
```
```js
const loading = document.getElementById("js-loading");
```

Reply:
Thank you!
Certainly I've seen the naming of the class "js -...".
I changed it because I thought it would be easier to see and understand.

commit: 152fa02